dvc/cli.py

```diff
@@ -204,6 +250,9 @@ def get_main_parser():
     for cmd in COMMANDS:
         cmd.add_parser(subparsers, parent_parser)
 
+    parser.cmd_choices = list(subparsers.choices.keys())
+    parser.cmd_choices.extend(help_action.option_strings)
```

Comment:
Functionally this works but it is just as strange as the line before. `--help/-h` aren't dvc commands but they function like arguments which is why they needed to be included in `cmd_choices`.